packages/hub/services/validators/scheduled-payment.ts

```diff
@@ -77,6 +76,12 @@ export default class ScheduledPaymentValidator {
       }
     }
 
+    if (!scheduledPayment.payAt && (!scheduledPayment.recurringDayOfMonth || !scheduledPayment.recurringUntil)) {
```

Comment:
In case `payAt` and `recurringUntil` are blank, and `recurringDayOfMonth` is present, then I think `recurringDayOfMonth` will have an error that says it's missing which is incorrect